compiler-cli/src/build.rs

```diff
@@ -13,7 +14,12 @@ use crate::{
     fs::{self, ConsoleWarningEmitter},
 };
 
-pub fn main(options: Options) -> Result<Package> {
+pub struct Built {
+    pub root_package: Package,
+    pub module_interfaces: im::HashMap<SmolStr, type_::Module>,
```

Comment:
This is an internal data structure and not to be exposed 🙏

Reply:
Same as my other comment.